src/locales/click-to-load/en/facebook.json

```diff
@@ -32,25 +32,29 @@
         "note": "Mouse-over description for the Facebook login button "
     },
     "buttonTextUnblockContent": {
-        "title": "Unblock Content",
+        "title": "Unblock Facebook Content",
         "note": "Button user would click on to unblock generic Facebook content"
     },
     "buttonTextUnblockComment": {
-        "title": "Unblock Comment",
+        "title": "Unblock Facebook Comment",
         "note": "Button user would click on to unblock a single Facebook comment"
     },
     "buttonTextUnblockComments": {
-        "title": "Unblock Comments",
+        "title": "Unblock Facebook Comments",
         "note": "Button user would click on to unblock multiple Facebook comments"
     },
     "buttonTextUnblockPost": {
-        "title": "Unblock Post",
+        "title": "Unblock Facebook Post",
         "note": "Button user would click on to unblock a Facebook post"
     },
     "buttonTextUnblockVideo": {
-        "title": "Unblock Video",
+        "title": "Unblock Facebook Video",
         "note": "Button user would click on to unblock a Facebook video"
     },
+    "buttonTextUnblockLogin": {
```

Comment:
`buttonTextUnblockLogin` doesn't appear to be used anywhere that I could find?

Reply:
You're right, it is not used now. But it will be used in the new Facebook login button I'm adding in this other [PR](https://github.com/duckduckgo/content-scope-scripts/pull/586/files#diff-72a00a07ebf3e836e03b9e806d9ca27f88eb9a0d772559e61c391ee1d764770bR628).